src/core/render-biblio.js

```diff
@@ -4,9 +4,33 @@
 import { addId } from "./utils";
 import hyperHTML from "hyperhtml";
 import { pub } from "./pubsubhub";
+import { lang as defaultLang } from "../core/l10n"
 
 export const name = "core/render-biblio";
 
+const localization_strings = {
```

Comment:
We can just call this.. 
```suggestion
const l10n = {
```